.gitignore

```
@@ -8,3 +8,8 @@ wwwroot/
 node_modules/
 *.user
 .pnpm-debug.log
+/.editorconfig
+/TestSolutions/Lombiq.Analyzers.*/.editorconfig
+/TestSolutions/Lombiq.Analyzers.*/AnalyzerViolations.cs
+/TestSolutions/Lombiq.Analyzers.PackageReference/IndirectPackageReference/AnalyzerViolations.cs
```

Combine these two lines to:
```suggestion
/TestSolutions/Lombiq.Analyzers.*/**/AnalyzerViolations.cs
```